packages/boxed-expression-component/src/expressions/DecisionTableExpression/DecisionTableExpression.tsx

```diff
@@ -895,11 +902,19 @@ export function DecisionTableExpression({
           case DecisionTableColumnType.OutputClause:
             const newOutputs = [...(prev.output ?? [])];
             newOutputs.splice(localIndexInsideGroup, 1);
-
+            const updatedOutputColumns = [
+              ...(newOutputs ?? []).map((outputColumn) => {
+                const outputCopy = { ...outputColumn };
+                if (newOutputs.length == 1) {
+                  outputCopy["@_name"] = undefined;
+                }
+                return outputCopy;
+              }),
+            ];
```

Comment:
I am not an expert in the `splice`, however, can it cause `newOutputs` to be again undefined? Can not we remove one ` ?? []` ? Also I think we should use `===` instead of `==` in typescript code.
```suggestion
            const updatedOutputColumns = [
              ...(newOutputs).map((outputColumn) => {
                const outputCopy = { ...outputColumn };
                if (newOutputs.length === 1) {
                  outputCopy["@_name"] = undefined;
                }
                return outputCopy;
              }),
            ];
```

Reply:
Also, I think we should bring more information. The variable name `updatedOutputColumns` will be difficult to understand in the future. From the variable, it is difficult to understand why we do update and what we update.

Could we try to bring more meaningful variable name? Or at least add comment that we explain update is done because of kie-issues#1466 ?

Reply:
Yeah i have updated and added the comment ,regarding ?? [] - it will be only used to perform map operation when newOutputs is null or undefined and it is to avoid any runtime errors and it won't impact anything.